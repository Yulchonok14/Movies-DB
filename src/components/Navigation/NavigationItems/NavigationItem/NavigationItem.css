.NavigationItem {
    margin: 10px 0;
    box-sizing: border-box;
    width: 100%;
    height: 100%;
    display: flex;
    align-items: center;
}

.NavigationItem a {
    text-decoration: none;
    height: 50%;
    box-sizing: border-box;
    display: block;
}

.NavigationItem img {
    height: 100%;
}

@media(min-width: 500px){
    .NavigationItem {
        margin: 0;
        width: auto;
    }

    .NavigationItem a {
        height: 33px;
        border-bottom: 4px solid transparent;
    }

    .NavigationItem a:hover,
    .NavigationItem a:active,
    .NavigationItem a.active {
        border-bottom: 4px solid #f65261;
    }
}