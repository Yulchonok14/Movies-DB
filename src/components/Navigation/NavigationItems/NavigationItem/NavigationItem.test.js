import React from 'react';

import { configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer'
import Magnifier from '../../../../assets/images/magnifier.png';

import NavigationItem from './NavigationItem';

configure({
    adapter: new Adapter()
});

describe('<NavigationItem/>', () => {

    it('NavigationItem renders correctly', () => {
        const tree = renderer
            .create(<NavigationItem link={'/'}><img src={Magnifier} alt="MagnifierSign"/></NavigationItem>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});