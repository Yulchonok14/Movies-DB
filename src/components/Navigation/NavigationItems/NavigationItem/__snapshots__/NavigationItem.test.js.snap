// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationItem/> NavigationItem renders correctly 1`] = `
<li>
  <a
    href="/"
  >
    <img
      alt="MagnifierSign"
      src="test-file-stub"
    />
  </a>
</li>
`;
